src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -81,20 +92,20 @@ protected override IRequestExecutionQueue<RazorRequestContext> ConstructRequestE
     protected override ILspServices ConstructLspServices()
     {
         var services = new ServiceCollection()
-            .AddOptions()
-            .AddLogging();
+            .AddOptions();
+
+        var loggerFactoryWrapper = new LoggerFactoryWrapper(_loggerFactory);
```

Comment:
could this have been MEF injected? but seems like an overkill since it is already a private class for RazorLanguageServer class

Reply:
Yes, but it would have had to be passed through from three layers up the stack, so not sure it's worth it